### PR TITLE
[MIG] migrate business_requirement_gap_analysis_task from 10.0 to 12.0

### PR DESCRIPTION
**Pull request by ruter-lyu**

### CODE REVIEW

**Review by elicoidal — COMMENTED** [review]
LGTM. Please prepare the README

**Review by JoJoJoJoJoJoJo — APPROVED** [review]
LGTM

File: business_requirement_gap_analysis_task/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (nikul-serpentcs) [inline business_requirement_gap_analysis_task/README.rst:3]: @ruter-lyu LGPL-3

File: business_requirement_gap_analysis_task/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+=====
+
+#. Create a new BR with a master project
+#. The button `Generate the Gap Analysis Task` will show
+#. Click the button and the gap analysis task will be created
+#. A new field `Gap Analysis Task` is added in the form view of the BR
+
+.. figure:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/222/10.0
```
Comment (nikul-serpentcs) [inline business_requirement_gap_analysis_task/README.rst:22]: 10/12

File: business_requirement_gap_analysis_task/wizard/create_gap_task.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <field name="priority">20</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field name="estimated_time"/>
+                </group>
+                <footer>
+                <button name="apply" string="OK" type="object"
+                        class="oe_highlight"/>
+                <button string="_Cancel" class="oe_link" special="cancel"/>
```
Comment (nikul-serpentcs) [inline business_requirement_gap_analysis_task/wizard/create_gap_task.xml:20]: string="Cancel"

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor Change

**Comment by pedrobaeza** [comment]
Closing as old and not green

**Pull request closed without merging**